plugins/cost-insights/src/utils/styles.ts

```diff
@@ -356,16 +380,33 @@ export const useTooltipStyles = makeStyles<CostInsightsTheme>(
   (theme: CostInsightsTheme) =>
     createStyles({
       tooltip: {
-        padding: theme.spacing(1.5),
         backgroundColor: theme.palette.tooltip.background,
         borderRadius: theme.shape.borderRadius,
         boxShadow: theme.shadows[1],
         color: theme.palette.tooltip.color,
         fontSize: theme.typography.fontSize,
+        width: 250,
+      },
+      actions: {
+        padding: theme.spacing(2),
+      },
+      header: {
+        padding: theme.spacing(2),
+      },
+      content: {
+        padding: theme.spacing(2),
       },
       lensIcon: {
         fontSize: `.75rem`,
       },
+      divider: {
+        /* TODO: tooltip divider color should be the inverse of theme. 
+        Dark mode should appear dark, light mode should appear light */
+        backgroundColor: darken(theme.palette.divider, 1),
```

Comment:
Haven't figured out a solution for this UI quirk.

Tooltip backgrounds appear as the inverse of the theme. Divider background color also appear as an inverse of the theme. In other words, tooltips and dividers both appear light in dark mode, and dark in light mode to create contrast.

Because the tooltip contains a divider, the divider needs to be the inverse of the tooltip color scheme to create contrast.

Reply:
I think you want MUI's `emphasize` function.